views/layout.pug

```diff
@@ -22,5 +22,3 @@ html
     div.footer-nav
 
     include footer
-    script(src="/javascripts/index.js")
```

Comment:
This was intentional since it isn't needed. Harman pointed this out to me as we were refactoring yesterday.